### PR TITLE
Add workaround to force Java 11 compatibility

### PR DESCRIPTION
**Pull request by jabrah**
* Add a compiler argument to the test runner to ignore illegal access exceptions
  * The `powermock` library used in some tests in this repository seems to have never been updated to support Java 11 properly. This workaround explicitly ignores the new exceptions.

### CODE REVIEW

**Comment by bbranan** [comment]
PR #172 should resolve the issue in the GH Action. I'd suggest waiting until that one is merged, then rebase this one on main and give it another go.

**Comment by jrmartino** [comment]
checked out this branch and ran mvn -U clean install
, the parent snapshot was pulled from sonatype. 

first time through i got an assertion error, looked like a timeout problem. ran this command a second time, and this time it sailed through.

**Review by jrmartino — APPROVED** [review]
my machine is a little slow, first time through i got something that looked like a timeout issue, but re-running the build succeeded. verified that the parent snapshot was pulled, and that i was running on java 11

**Pull request merged**
